scripts/adopt.js

```diff
@@ -0,0 +1,60 @@
+function disableButton(id) {
+	document.getElementById(id).disabled = true;
```

Comment:
Nice concept to keep the user from pressing adopt multiple times on the same dog!